lib/active_interaction/filter.rb

```diff
@@ -205,12 +209,14 @@ def describe(value)
       "(Object doesn't support #inspect)"
     end
 
+    # @param context [Base, nil]
```

Comment:
It never is, but it can be. When we call `default` in `eager_evaluate_default`, we don't have an interaction to pass in. 

If the default is a `Proc`, then the `context` is never `nil`. That's hard to express, though.